include/crocoddyl/core/activation-base.hpp

```diff
@@ -12,12 +12,13 @@
 #include <stdexcept>
 #include <boost/shared_ptr.hpp>
 #include <boost/make_shared.hpp>
-#include <boost/core/demangle.hpp>
 
 #include "crocoddyl/core/fwd.hpp"
 #include "crocoddyl/core/mathbase.hpp"
 #include "crocoddyl/core/utils/to-string.hpp"
 
+#include <boost/core/demangle.hpp>
```

Comment:
Why do we need to include this header after Crocoddyl stuff?

The common rule is to include external headers first.

Reply:
Sure